src/components/navbar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import logo from '../planet.png';
+
+export default function NavBar() {
+  return (
+    <>
+      <header className="navigation">
+        <div className="header">
+          <img src={logo} alt="logo" />
+          <h1>Space Travelers&apos; Hub</h1>
+        </div>
+        <nav className="navBar">
+          <NavLink to="/">Rockets</NavLink>
+          <NavLink to="/missions">Missions</NavLink>
+          <NavLink to="/dragons">Dragons</NavLink>
+          <NavLink to="/myprofile">MyProfile</NavLink>
+        </nav>
```

Comment:
- [ ] The active page should be indicated with an underline, currently, the underline appears when you click on the navigation links but please make it such that the active page has an underline not just only when you click on it.

Reply:
Done.